keypom/keypom-airfoil
# chore: all drops drop manager issues

# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #134

# How to test
Show steps to replicate and test the feature/fixes in this PR

For EXPORTS CSV
1. go to https://andrel-134-all-drops-drop-ma.keypom-airfoil.pages.dev/
2. connect to your NEAR wallet
3. go to My Drops
4. Select any drops with keys
5. Click on `Export .CSV`
6. ALL KEYS in the drops will now be exported into .csv file

For MASTER_KEY
1. go to https://andrel-134-all-drops-drop-ma.keypom-airfoil.pages.dev/
2. connect to your NEAR wallet
3. go to My Drops
4. set a wrong master key
5. select any drops
6. you should see a modal to inform wrong master key
7. set the correct master key
8. refresh, modal doesnt show anymore

For SCANNED NUMBER: (tested on low-tier throttling mobile device on 1000keys)
1. go to https://andrel-134-all-drops-drop-ma.keypom-airfoil.pages.dev/
2. connect to your NEAR wallet
3. go to My Drops
4. Click on Ticket Drop with TONNES of keys
5. wait few seconds for `Scanned number` to load
# Screenshots/Screen Recording of Implementation
see comment
